uaclient/jobs/update_messaging.py

```diff
@@ -71,6 +71,14 @@ def get_contract_expiry_status(
     grace_period = defaults.CONTRACT_EXPIRY_GRACE_PERIOD_DAYS
     pending_expiry = defaults.CONTRACT_EXPIRY_PENDING_DAYS
     remaining_days = cfg.machine_token_file.contract_remaining_days
+
+    # if unknown assume the worst
+    if remaining_days is None:
+        logging.warning(
+            "contract effectiveTo date is null - assuming it is expired"
+        )
+        return ContractExpiryStatus.EXPIRED, -grace_period
```

Comment:
@orndorffgrant should we also change that logic for fix too ?
https://github.com/canonical/ubuntu-advantage-client/blob/main/uaclient/security.py#L1292

Just to be consistent here

Reply:
good catch! I'll change it

Reply:
pushed!